src/test/java/org/takes/http/BkParallelTest.java

```diff
@@ -80,12 +83,16 @@ public Response act(final Request req)
             }
         };
         new Thread(
+            // @checkstyle AnonInnerLengthCheck (23 lines)
             new Runnable() {
                 @Override
                 public void run() {
                     try {
                         new FtBasic(
-                            new BkParallel(new BkBasic(take)),
+                            new BkParallel(
+                                new BkBasic(take),
+                                count
```

Comment:
@yegor256 The idea is of using the same amount of threads for `BkParallel` and for requests. Previously used `availableProcessors()` and could be not enough.
And min between 3 and availableProcessors (lines 65..68) is to be sure that it have enough threads to make all request and can respond it.